elasticdl/python/common/tensor.py

```diff
@@ -47,12 +47,46 @@ def deserialize_tensor_pb(tensor_pb, tensor):
         )
     tensor.set(
         np.ndarray(shape=tensor_pb.dim, dtype=dtype, buffer=tensor_pb.content),
```

Comment:
Add the parameter name here such as 
values = np.ndarray(shape=tensor_pb.dim, dtype=dtype, buffer=tensor_pb.content),
indices= np.array(tensor_pb.indices) if tensor_pb.indices else None

will be more explicit

Reply:
Done.